### ISSUE TITLE
A9G won;t power ON

### ISSUE THREAD

**Issue by medida** [Ai-Thinker-Open/GPRS_C_SDK#347]
Hi

I'm using the SDK V2112

I have 2 A9G boards setup as i'm experimenting for a new project.
Both of them no longer respond to any power?
I can see in coolwatcher - that when i press the reset key i get 'Detected event: 0xffffffff'

but they don;t seem to be running any code or any LEDS flashing.

Any help much appreciated - either I have broken both of them! or they are stuck in some weird boot mode...
I did have them running a demo project - but stopped / unwired to insert a sim card.

Please help. 
Thanks

**Comment by medida** [comment]
.... After restart everything - and sending the project to the device again - it seems its no doing something....
Very strange.

Is there any reason why the device would have LOST its coding

**Comment by pulkin** [comment]
Yes, sometimes you may hang the communication. For me (linux) it happens if I start capturing events in `coolwatcher`. I just restart it.

**Comment by BrickStacking** [comment]
Hi, I have the same problem as you. Now my A9G can't flash new code or active tracer. Did you solve the problem? please help !!

**Comment by medida** [comment]
from memory i think i just held down the power buttons and tried to PUSH the new code at the right moment.

However i currently have a unit on my desk which is not responding to anything - i think its dead!

**Comment by rahulmr8222** [comment]
@medida

**Comment by rahulmr8222** [comment]
Facing problem flashing micropython into A9G,  can u help me out?

**Comment by medida** [comment]
> Facing problem flashing micropython into A9G, can u help me out?

Sorry - I've not used this since 2020 as i gave up on the hardware because it was too power intensive

**Comment by rahulmr8222** [comment]
Ok thankyou bro.

**Comment by denicsdevices** [comment]
In my case, just changing the USB - TTL adaptor worked. My board was custom built though, and there I forgot to pull down Pin9-PWR_KEY to low with 10K resistor, so it was not booting up at first. After doing the required changes the board booted correctly but showed same behavior like o/p mentioned. I tired few different adaptors and one with FTDI Chip worked for me. CP21xx chip adaptor is also working now after flashing the A9G. BTW, make sure you are connecting the adaptor rx/tx to HST UART port and not to UART1 or 2. Also it is recommended to provide 4.2V at VBAT while flashing. 

Another thing is you need to hold down PWR button on the Pudding module at least for 3-4 seconds to boot up the board completely, without that nothing will happen
Good luck

**Comment by denicsdevices** [comment]
> > Facing problem flashing micropython into A9G, can u help me out?
> 
> Sorry - I've not used this since 2020 as i gave up on the hardware because it was too power intensive

It is not that power intensive. the module will draw at max 10mA* while flashing or doing communication, At idle it will draw 2mA*

*checked on my custom built board. using switching regulator AP3402 for 4.2V to stepdown 6V (4x AA Batteries @ 1.5v). The tests conducted with lab grade power supply connected to battery input port.

**Comment by Unisolgr14** [comment]
@denicsdevices My A9G board is now not powering up (it used to work all the AT commands). Now no leds ,no AT command response. So,I bought a new TTL convertor but still the A9G board does not work.

**Comment by skyelectricir** [comment]
> [@denicsdevices](https://github.com/denicsdevices) My A9G board is now not powering up (it used to work all the AT commands). Now no leds ,no AT command response. So,I bought a new TTL convertor but still the A9G board does not work.

Hello. Your problem is solved? I have the same problem :/
